source/jsonnet/england-wales/lib/transforms.libsonnet

```diff
@@ -0,0 +1,96 @@
+local formatPersonName(sameNameTransform=false) = (
+
+  local includeMiddleNames = if sameNameTransform then { source: 'previous_transform' } else false;
```

Comment:
When I suggested we should use the `format_name` transform in place of `concatenate_list`, I hadn't realised that `middle_names` and `include_middle_names` were required properties. I assumed we could omit them. This now means that in a lot of places we are looking up the middle names answer to throw it away. We should either make them optional (meaning runner and validator changes), or revert back to the concatenate list transform.

Reply:
Looks like concatenate list transform is coming back, the plan was to spend minimal time on this and not change the other repos, reverting will mean I don't need that validator change